mmocr/models/textrecog/decoders/aster_decoder.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn.functional as F
+
+from mmocr.models.builder import DECODERS
+from mmocr.models.textrecog.layers import attention_aster_layer
+from .base_decoder import BaseDecoder
+
+
+@DECODERS.register_module()
+class ASTERDecoder(BaseDecoder):
```

Comment:
- ASTER uses the Attention decoder, and I believe there should be a corresponding implementation in mmocr. 
- @gaotongxiao [SequenceAttentionDecoder](https://github.com/open-mmlab/mmocr/blob/52c95ccaac/mmocr/models/textrecog/decoders/sequence_attention_decoder.py) seems to be a generic Attention decoder? MMOCR has so many Attention-based algorithms, there should be a generic Attention decoder, just like our recently reviewed Master, there is a generic transformer decoder.